Dockerfile.ubi

```diff
@@ -1,5 +1,5 @@
 ## Global Args #################################################################
-ARG BASE_UBI_IMAGE_TAG=9.4
+ARG BASE_UBI_IMAGE_TAG=9.4-1227
```

Comment:
This is not required, using `9.4` will automatically fetch the latest available version.

Reply:
@dtrifiro IMO it's best to use the specific version tags. Otherwise you have no idea what was actually pulled .. the image contents just depends on when the build happened to run and it isn't reproducible.

Reply:
We want to be able to clearly say yes this build is based on this specific UBI image, without having to dig through CI logs or inspect the internal of the built image.

Reply:
I understand your reproducibility concern, but unless we configure something like renovate (https://docs.renovatebot.com) to automatically open PRs whenever a new UBI image is available, I think that it'd be better to avoid the overhead of having to bump the version manually as it's something that's easy to be forgotten